core/controllers/acl_decorators.py

```diff
@@ -1883,20 +1875,8 @@ def test_can_edit_skill(self, skill_id, **kwargs):
         if not self.user_id:
             raise base.UserFacingExceptions.NotLoggedInException
 
-        skill_rights = skill_services.get_skill_rights(
-            skill_id, strict=False)
-        if skill_rights is None:
-            raise base.UserFacingExceptions.PageNotFoundException
-
         if role_services.ACTION_EDIT_PUBLIC_SKILLS in self.user.actions:
```

Comment:
@aks681 Do we still have Public/private skill terminology?

Reply:
We don't have any split between them now. All skills are public.
Shall I rename this in a follow up PR as well?

Reply:
Renamed in #8363